### PR TITLE
Remove animation for transparent modals

### LINKED ISSUE #95: transparentCard: true doesn't work on Android
It works perfectly on iOS, but on Android the screen under modal is just being hidden.
Reproduce: https://snack.expo.io/S1ZKx1VvV

### PR DESCRIPTION
**Pull request by todorone**
Fixes #95

### CODE REVIEW

File: src/views/StackView/StackViewTransitionConfigs.tsx [hunk 1]
```diff
@@ -78,8 +78,14 @@ const NoAnimation = {
 function defaultTransitionConfig(
   transitionProps: TransitionProps,
   prevTransitionProps?: TransitionProps,
-  isModal?: boolean
+  isModal?: boolean,
+  isTransparent?: boolean
 ): TransitionConfig {
+  // If modal is not full-screen, user is responsible to implement animation
+  if (isModal && isTransparent) {
```
Comment (JustinKasad) [inline src/views/StackView/StackViewTransitionConfigs.tsx]: Should we add an `Platform.OS === 'android'` check here since the animation works well on iOS?

Comment (todorone) [reply src/views/StackView/StackViewTransitionConfigs.tsx]: As discussed with @brentvatne on https://github.com/react-navigation/stack/issues/95#issuecomment-475732336 we decided to remove animation to unify both platforms...

**Comment by todorone** [comment]
@satya164 Whatever You say. But before opening PR, I've received approval from @brentvatne that it's not breaking change - https://github.com/react-navigation/stack/issues/95#issuecomment-475732336

**Comment by satya164** [comment]
No need to be rude.

This PR had been open for 5 months and the feature has been there in the stable version for a long time. A lot of people are already using it and changing the behavior this late will mean they'll have to change their code.

Also,  we already have another major version releasing soon with many breaking changes, no reason to introduce a breaking change in minor version.

**Comment by todorone** [comment]
Please accept my apologies. Good luck!

**Pull request closed without merging**